charts/pega/templates/pega-environment-config.yaml

```diff
@@ -10,6 +10,10 @@ data:
   DB_TYPE: {{ .Values.global.jdbc.dbType }}
   # JDBC URL of the DB where Pega is installed
   JDBC_URL: {{ .Values.global.jdbc.url }}
+{{- if .Values.global.jdbc.readerurl }}
+  # URI that the JDBC driver can be downloaded from
```

Comment:
Looks like a copy/paste remnant -- this is the read only URL not the driver UIR, right?

Reply:
Good catch, fixed this